docs/manuals/source/Appendix/OperatingSystems.rst

```diff
@@ -54,29 +54,47 @@ Additionally, the following table gives an overview on the available package ver
 Linux
 -----
 
-.. _section-packages:
+.. _section-UniversalLinuxClient:
 
-Packages for the different Linux platforms
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Universal Linux Client
+~~~~~~~~~~~~~~~~~~~~~~
 
-The following tables summarize the packages that are available for individual Linux platforms.
-Package names that do not contain the word **bareos** are required third party packages, provided by the Bareos project.
+The Bareos project provides packages
+for the current releases of all major Linux distributions.
+In order to support even more platforms
+Bareos :sinceVersion:`21.0.0: Universal Linux Client (ULC)`
+provides the so called **Universal Linux Client (ULC)**.
 
-For general information about the packages, see :ref:`section-BareosPackages`.
+The Universal Linux Client is a |fd|,
+built in a way to have minimal dependencies to other libraries.
 
-.. include:: /include/autogenerated/bareos-packages-table-redhat.rst.inc
+.. note::
 
-.. include:: /include/autogenerated/bareos-packages-table-fedora.rst.inc
+   The **Universal Linux Client** depends on the **OpenSSL** library
+   of the host in order to utilize security updates for this library.
 
-.. include:: /include/autogenerated/bareos-packages-table-suse.rst.inc
+It incorperates all functionality for normal backup and restore operations,
+however it has only limited plugin support.
 
-.. include:: /include/autogenerated/bareos-packages-table-opensuse.rst.inc
+Currently it is provided as a Debian package.
+However, it it planed to provide it also in other formats.
 
-.. include:: /include/autogenerated/bareos-packages-table-debian.rst.inc
+The ULC components come in extra repositories (names starting with **ULC_**)
+at https://download.bareos.org/bareos/release/
+and consists of only a single package per platform.
```

Comment:
This is not exactly true, as we also have the debug package (see  contents of https://download.bareos.org/bareos/experimental/nightly/ULC_deb_OpenSSL_1.1/amd64/)

Reply:
Add remark about debug packages.